test/integration/spec/rest.js

```diff
@@ -20,26 +20,48 @@ const {
 } = require('../../lib/util');
 
 const connect = require('../../../lib/connect');
+const { RoomMaxParticipantsExceededError } = require('../../../lib/util/twilio-video-errors');
 
 // TODO(mmalavalli): Use twilio-node to call Track Subscription REST APIs.
 
-function subscribedTracks(publication, room, trackAction) {
-  const { localParticipant, sid } = room;
+function post(resource, data) {
   return new Promise((resolve, reject) => {
     const request = https.request({
       auth: `${apiKeySid}:${apiKeySecret}`,
       headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
       hostname: 'video.twilio.com',
       method: 'POST',
-      path: `/v1/Rooms/${sid}/Participants/${localParticipant.sid}/SubscribedTracks`
+      path: resource
     }, response => {
       response.on('data', () => {});
       response.on('end', resolve);
     });
     request.once('error', reject);
-    request.write(`Track=${publication.trackSid}&Status=${trackAction}`);
+    request.write(Object.keys(data).map(key => `${key}=${data[key]}`).join('&'));
     request.end();
   });
+
+}
+
+function completeRoom(name) {
+  return post(`/v1/Rooms/${name}`, {
+    Status: 'completed'
+  });
+}
```

Comment:
IMO, we should start using twilio-node. See how we can complete a Room [here](https://github.com/twilio/twilio-node/blob/master/lib/rest/video/v1/room.js#L690).

Reply:
@markandrus `twilio-node` does not seem to work in the browser due to node-specific dependencies.